app/build.gradle.kts

```diff
@@ -44,6 +44,8 @@ dependencies {
     implementation(libs.androidx.activity)
     implementation(libs.androidx.constraintlayout)
     testImplementation(libs.junit)
+    testImplementation("org.jetbrains.kotlin:kotlin-test")
+    testImplementation("org.robolectric:robolectric:4.9")
```

Comment:
Why did you add robolectric to the project here?
Same question about kotlin-test

Reply:
yes i will delete them, i just had some issue with running tests, and tried out many ways and eventually i fixed that with installing a new gradle